resources/conf/config.toml

```diff
@@ -217,6 +217,13 @@ allowCredentials = false
     # Certificate Filepath within enforcer
     certificateFilePath = "/home/wso2/security/truststore/mg.pem"
 
+# Issuer 3
```

Comment:
Shall we add this to docker with APIM, k8s and k8s with APIM configs as well

Reply:
+1

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1905/commits/451a8ae5f5485698015687efbd5490211ac74951